### PR TITLE
Release `v0.9.0`

### PR DESCRIPTION
**Pull request by ruizb**
This PR releases the version 0.9.0 of the action, including:
* https://github.com/DataDog/synthetics-ci-github-action/pull/107

There are no changes in the documentation.

### CODE REVIEW

**Review by kalinichenko — APPROVED** [review]

**Comment by ruizb** [comment]
I'm going to merge without waiting for documentation folks' approval, as no docs were updated and this issue is kind of hot 🔥

**Pull request merged**
